lib/ddtrace/opentracer/tracer.rb

```diff
@@ -1,6 +1,22 @@
+require 'ddtrace/tracer'
+
 module Datadog
   module OpenTracer
+    # OpenTracing adapter for Datadog::Tracer
     class Tracer < ::OpenTracing::Tracer
+      extend Forwardable
+
+      attr_reader \
+        :datadog_tracer
+
+      def_delegators \
+        :datadog_tracer,
+        :configure
+
+      def initialize(options = {})
+        super()
+        @datadog_tracer = Datadog::Tracer.new(options)
```

Comment:
Is this configuration immutable? If `options` changes at run-time will we get undefined behaviour?

Reply:
The `options` Hash is not immutable. However, if you dig into the code, you'll see it doesn't store the Hash, but copies the values out of the hash and stores them in different variables. So it acts immutable, in that sense.